src/cli/run_impl.rs

```diff
@@ -61,6 +61,7 @@ pub fn run(setup: Setup) {
         chain_controller.clone(),
         shared.clone(),
         tx_pool_controller.clone(),
+        synchronizer.peers(),
```

Comment:
I have notice Peers have been moved out from synchronizer module, here is better to move out the initialize of Peer from Synchronizer struct to keep logical consistent.